docs/limitations.md

```diff
@@ -8,7 +8,9 @@ enterprise: 'yes'
 *   DC/OS Spark only supports submitting jars and Python scripts. It
 does not support R.
 
-*   Mesosphere does not provide support for Spark app development.
+*   Mesosphere does not provide support for Spark app development,
```

Comment:
Eh, I disagree with keith.  I don't think there's any ambiguity in what "Spark app development" mean.  But you can merge this if you like.